accounting/accounting_cli.py

```diff
@@ -98,6 +137,9 @@ def main():
         print("Unable to open database file")
         sys.exit(-1)
 
+    # set path for output file
+    output_file = args.output_file if args.output_file else "job_records.csv"
```

Comment:
My initial instinct is that this command should write to stdout by default rather than a file.  Do the Slurm utilities write to a file by default?

Reply:
You're right - that was something I forgot to implement. I went ahead and changed this so that if no output file is specified, it should just write to stdout, in the same format as it would have if it were written to a file:

```
[fluxuser@137e364cd768 ~]$ flux account -p /var/lib/flux/jobs.db by-user fluxuser
UserID|Username|JobID|T_Submit|T_Run|T_Inactive|Nodes|R
2001|fluxuser|429932937216|1597722545.7235024|1597722545.7498028|1597722545.83819|1|{"version":1,"execution":{"R_lite":[{"rank":"0","children":{"core":"0-2"}}]}}
2001|fluxuser|408659427328|1597722544.4558651|1597722544.4830627|1597722544.585075|1|{"version":1,"execution":{"R_lite":[{"rank":"0","children":{"core":"0-2"}}]}}
2001|fluxuser|340007059456|1597722540.363928|1597722540.387967|1597722540.4699485|1|{"version":1,"execution":{"R_lite":[{"rank":"0","children":{"core":"0-1"}}]}}
2001|fluxuser|319857623040|1597722539.162982|1597722539.1850276|1597722539.2743495|1|{"version":1,"execution":{"R_lite":[{"rank":"0","children":{"core":"0-1"}}]}}
2001|fluxuser|256154533888|1597722535.3659017|1597722535.3874228|1597722535.4645026|1|{"version":1,"execution":{"R_lite":[{"rank":"0","children":{"core":"0"}}]}}
2001|fluxuser|241961009152|1597722534.5195136|1597722534.5461538|1597722534.6319313|1|{"version":1,"execution":{"R_lite":[{"rank":"0","children":{"core":"0"}}]}}
```